packages/manager/src/components/AreaChart/AreaChart.tsx

```diff
@@ -228,7 +228,7 @@ export const AreaChart = (props: AreaChartProps) => {
   };
 
   return (
-    <>
+    <Box data-testid="areachart-wrapper">
```

Comment:
@nikhagra-akamai  / @agorthi-akamai  / @kmuddapo , since we added a box in common component, did we inform / check with cloud manager team

Reply:
we will inform them tomorrow